client/components/dialog/style.scss

```diff
@@ -27,6 +27,10 @@
 	}
 }
 
+.dialog__backdrop.is-hidden {
+	background-color: rgba( white, 0 );
```

Comment:
Minor nitpick: we could obtain the same outcome with `background-color: transparent`.

Reply:
Thanks! Updated in follow up PR https://github.com/Automattic/wp-calypso/pull/24797.